spark/pom.xml

```diff
@@ -171,13 +171,14 @@
             </goals>
             <configuration>
               <descriptors>
-                <descriptor>src/main/assembly/job.xml</descriptor>
+                <descriptor>../spark/src/main/assembly/job.xml</descriptor>
```

Comment:
i still don't understand why canonical path is replaced with non-canonical path here.